packages/web/src/app/doc-pages/about/tutorial/vue-tutorial/vue-tutorial.component.html

```diff
@@ -0,0 +1,88 @@
+  <li class="e-mt-24">
+    <h3 class="e-title-xsmall e-m-0">Start using classes</h3>
+    <p class="e-my-8">
+      Choose between multiple
+      <a routerLink="/tools/utility-classes" class="e-link e-link--inline">brand utilities</a> and
+      <a routerLink="/components" class="e-link e-link--inline">components</a>. The components have multiple
+      variants and options to fit different needs.
+    </p>
+    <app-code-viewer [tabs]="['HTML']" fileName="App.vue" [activeCode]="codeStep3" hideTabs />
+  </li>
+  <li class="e-mt-24">
+    <h3 class="e-title-xsmall e-m-0">Add CSS-reset<span class="e-tag e-ml-8">Optional</span></h3>
+    <p class="e-mt-8">
+      Import the Elvis CSS reset if you wish to apply a generic CSS reset to your application.
+    </p>
+    <p class="e-my-8 e-text-md">
+      <strong
+        >Be aware that you may break styles in your app if you import this in an existing application, since
+        you may rely on browser defaults without knowing it. This is why this import is separate from the
+        required Elvis import.</strong
+      >
+    </p>
+    <app-code-viewer [tabs]="['Typescript']" fileName="main.ts" [activeCode]="codeStep4" hideTabs />
+  </li>
+</ol>
+
+<!--ADVANCED TUTORIAL-->
+<h3 class="e-title-small e-mt-64">Add an advanced component</h3>
+<ol class="e-list e-list--numbers e-mt-40">
+  <li>
+    <h3 class="e-title-xsmall e-m-0">Install component</h3>
+    <p class="e-mt-8">
+      Install the
+      <a routerLink="/components/datepicker" class="e-link e-link--inline">datepicker component.</a>
+    </p>
+    <p class="e-my-8 e-text-md">
+      <strong
+        >The packages act like any other external dependency you install and update in your project.
+      </strong>
+    </p>
+    <app-code-viewer [tabs]="['Bash']" fileName="Terminal" [activeCode]="codeStepAdvanced1" hideTabs />
+  </li>
+  <li class="e-mt-24">
+    <h3 class="e-title-xsmall e-m-0">Add imports</h3>
+    <p class="e-my-8">Import the component.</p>
+    <app-code-viewer [tabs]="['Typescript']" fileName="main.ts" [activeCode]="codeStepAdvanced2" hideTabs />
+  </li>
+  <li class="e-mt-24">
+    <h3 class="e-title-xsmall e-m-0">Start using advanced components</h3>
+    <p class="e-my-8">
+      Choose between several different advanced components and customize them according to your needs by using
+      the available props.
+    </p>
+    <app-code-viewer [tabs]="['HTML']" fileName="App.vue" [activeCode]="codeStepAdvanced3" hideTabs />
+  </li>
+  <li class="e-mt-24">
+    <h3 class="e-title-xsmall e-m-0">Using props and events</h3>
+    <p class="e-my-8">
+      Try adding the fullWidth prop to see the datepicker take up full width of available space. Try adding
+      valueOnChange event to see the events coming in when the datepicker changes. The
+      <a routerLink="/components/datepicker" fragment="API" class="e-link e-link--inline">datepicker API</a>
+      lists all the properties and events available for datepicker.
```

Comment:
```suggestion
      Try adding the <code>fullWidth</code> prop to see the datepicker take up full width of available space. Try adding
      the <code>valueOnChange</code>-event to see the events coming in when the datepicker changes. The
      <a routerLink="/components/datepicker" fragment="API" class="e-link e-link--inline">datepicker API</a>
      lists all the properties and events available for datepicker.
```

Ensure that all these are alike across all tutorials 😄